Description: Remove an option for compatibility with mpl 3.2.1
 With matplotlib 3.2.1 this option leads to the error
 "ValueError: None is not a valid value for spacing; supported values are 'uniform', 'proportional'"
 during docbuild.
Author: Tobias Hansen <thansen@debian.org>

--- a/sage/src/sage/plot/contour_plot.py
+++ b/sage/src/sage/plot/contour_plot.py
@@ -223,7 +223,7 @@
                 cb.add_lines(CS)
 
 
-@suboptions('colorbar', orientation='vertical', format=None, spacing=None)
+@suboptions('colorbar', orientation='vertical', format=None)
 @suboptions('label', fontsize=9, colors='blue', inline=None, inline_spacing=3,
             fmt="%1.2f")
 @options(plot_points=100, fill=True, contours=None, linewidths=None,
